cluster-service/deploy/openshift-templates/arohcp-service-template.yml

```
@@ -256,6 +268,26 @@ objects:
       usePodIdentity: "false"
     provider: azure
 
+- apiVersion: secrets-store.csi.x-k8s.io/v1
+  kind: SecretProviderClass
+  metadata:
+    name: arm-identity
+    namespace: ${NAMESPACE}
+  spec:
+    parameters:
+      clientID: ${ARM_HELPER_IDENTITY_CLIENT_ID}
+      cloudName: AzurePublicCloud
+      keyvaultName: ${ARM_HELPER_IDENTITY_KEYVAULT_NAME}
+      objects: |-
+        array:
+          - |
+            objectName: ${ARM_HELPER_IDENTITY_CERT_NAME}
+            objectType: secret
+            objectAlias: armHelperIndentityCertificateBundlePath
```

```suggestion
            objectAlias: armHelperIndentityCertificateBundle
```
This is a Cert Bundle and not a Path.